makefile.shade

```diff
@@ -5,16 +5,6 @@ k-standard-goals
 
 default DOTNET_ARGUMENT='-v '
 
-#build-compile target='compile'
```

Comment:
Korebuild already builds the projects in `src`, and the stuff under `experimental` is not necessary and can be built by whoever wants to work with it. Also, @natemcmaster is trying to change Korebuild to build the solution file instead of enumerating projects like we do now, which will eventually build the projects in `experimental` as part of the full build again.

We want to remove as much as possible from makefiles.